docs/compatible_microcontrollers.md

```diff
@@ -62,6 +62,55 @@ You can also use any ARM chip with USB that [ChibiOS](https://www.chibios.org) s
  * [MK66FX1M0](https://www.nxp.com/products/processors-and-microcontrollers/arm-microcontrollers/general-purpose-mcus/k-series-cortex-m4/k6x-ethernet/kinetis-k66-180-mhz-dual-high-speed-full-speed-usbs-2mb-flash-microcontrollers-mcus-based-on-arm-cortex-m4-core:K66_180)
    * PJRC Teensy 3.6
 
+### Raspberry PI RP2040
+
+The following drivers are currently supported on RP2040 MCUs.
+
+| System           | Support                                       |
+| ---------------- | --------------------------------------------- |
+| ADC driver       | Support planed                                |
```

Comment:
"planed" is misspelled - it should be "planned" :)

Reply:
`planed` also on line 72(Audio), 77(EEPROM), and 79(UART)

Reply:
Good catch, changed these